### ISSUE TITLE
react-native-reanimated v3 drops support for code this package uses (fixed)

### ISSUE THREAD

**Issue by Ario99** [alexZajac/react-native-skeleton-content#94]
**Describe the bug**
react-native-reanimated dropped support for api v1 which this repo depended on. The fix I have provided is a modification of @phanghos's code in their [issue](https://github.com/alexZajac/react-native-skeleton-content-nonexpo/pull/67) for the nonexpo version of this package. I have patched this issue with [patch-package](https://www.npmjs.com/package/patch-package) for v1.0.28. I used this patch and some resolutions to fix the issue for myself.

### Patch
**Resolutions**
- "react-native-reanimated": "~3.3.0"
- "expo-linear-gradient": "~12.3.0"
- "react-native-redash": "^18.1.0

**Here is the diff that fixed this problem for me**
[react-native-skeleton-content+1.0.28.patch](https://github.com/alexZajac/react-native-skeleton-content/files/14030924/react-native-skeleton-content%2B1.0.28.patch)

**To Reproduce**
You will run into the same problem upon upgrading to a newer version of the expo sdk. I am running this currently on expo sdk 49.

**Expected behavior**
Animations to play where they were previously working.

**Smartphone (please complete the following information):**
 - Device: SM-G991U1 (Galaxy S21 5G)
 - OS: Android v13, One UI v5.1
 - Package version: 1.0.28

**Additional context**
I understand that this issue may not be fixed with the info I provided for an indefinite period of time. The maintainer is predisposed so in the mean time I hope I have provided the information others may need so they will not have to jump through the same hoops I did. I definitely did learn a lot in the process. If you have any questions, I hope I will be able to answer in a timely matter.

Also it seems if you don't want to mess with `patch-package` to fix this. You may be able to use [this](https://www.npmjs.com/package/react-native-reanimated-skeleton) package in the mean time until the [pull request](https://github.com/alexZajac/react-native-skeleton-content/pull/77#issuecomment-1708760166) for the changes contained within @marcuzgabriel's are made. I'm not for sure if I tried their package previously but it may be more convenient for you than what I have given here.

**Comment by AnushavanGhulyan-melon** [comment]
This is the updated version.
https://github.com/marcuzgabriel/react-native-reanimated-skeleton

**Comment by Engazan** [comment]
> This is the updated version. https://github.com/marcuzgabriel/react-native-reanimated-skeleton

nah, its not for expo, it uses ``react-native-linear-gradient`` ... and not ``expo-linear-gradient`` ...

**Comment by marcuzgabriel** [comment]
@Engazan. Isn't it possible to switch out expo-linear-gradient package with react-native-linear-gradient or does expo not support that?

**Comment by Engazan** [comment]
> @Engazan. Isn't it possible to switch out expo-linear-gradient package with react-native-linear-gradient or does expo not support that?

I m not sure but it contains iOS and android folders witch almost always mesn that you need "config plugin" / prebuild (dev client) or eject (almost useless nowdays)

But when i tried it i got errors With can be solved only With "pod install" or config plugin :/ 

But for this project i cant do that, client doesnt want that

But there is "react-native-shimmer-placeholder" that works fine With expo and it only need minimum changes its almost same kind of, and it has driber option so you can choose what you want use (expo or something else ) witch is great

**Comment by marcuzgabriel** [comment]
Good you found an alternative 🥇 . I will try have a look on why react-native-linear-gradient won't work with expo and the other way around why expo-linear-gradient won't work with my library.

**Comment by Ario99** [comment]
I ended up switching to [this package](https://www.npmjs.com/package/react-native-reanimated-skeleton).

Kudo's to @marcuzgabriel for his easy installation guide for expo as well as an alternative.
